libs/components/tabs/src/lib/modules/tabs/tab.component.ts

```diff
@@ -122,21 +122,21 @@ export class SkyTabComponent implements OnChanges, OnDestroy {
    * If not defined, the identifier is set to the position of the tab on load, starting with `0`.
    */
   @Input()
-  public set tabIndex(value: SkyTabIndex | undefined) {
+  public set tabIndexValue(value: SkyTabIndex | undefined) {
```

Comment:
The docs say this is a "unique identifier". To get away from the `tabindex` concept entirely (and avoid future confusion), could we go with `uniqueKey`, `uniqueId`, `tabIdentifier` or something similar?

On the other hand, I'm not especially married to changing it from `tabIndexValue` either since the "tab index" phraseology is used all over the place in the code (e.g. SkyTabsetService.tabIndexesEqual`). I could go either way; up to you.

Reply:
I think the documentation is misleading. Each tab is assigned a tab index based on the order in which it appears in the tabset, so it's like a tab index in that sense, but the consumer can specify any value they want. I also found that if you specify a tab index that is automatically assigned to another tab based on its position, you get collisions and unexpected behavior like multiple tab panels displaying at once. I avoided that can of worms by making the minimal changes to fix the accessibility issue; we can always rename this again with a similar schematic if we come up with a better solution later.